src/util/dev/templates/error.jst

```diff
@@ -0,0 +1,9 @@
+<main>
+  <p class="devinfo-container">
+    <span class="error-code"><strong>{{= it.http_status_code }}</strong>: {{! it.http_status_description }}</span>
+    {{? it.error_code }}
+    <span class="devinfo-line">Code: <code>{{! it.error_code }}</code></span>
+    {{?}}
+    <span class="devinfo-line">ID: <code>{{! it.now_id }}</code>
+  </p>
+</main>
```

Comment:
Could these templates be shared with now-proxy instead of copy/pasta? Perhaps a separate npm package? 🤓

Reply:
Unfortunately not, they use a different format so these were manually converted to doT.js format. After doing it manually, I was considering writing a (brittle) converter for future use, but since these were already done manually I felt there was no need to do that now.